tests/unit/utils/test_exit_codes.py

```
@@ -0,0 +1,70 @@
+    assert_that(total_fixed).is_equal_to(2)
+    assert_that(total_remaining).is_equal_to(1)
+
+
+def test_aggregate_excludes_skipped_tools() -> None:
+    """Skipped tools contribute nothing to any total."""
+    results = [
+        ToolResult(name="ruff", success=False, issues_count=4),
+        ToolResult(name="black", skipped=True, skip_reason="missing", issues_count=0),
+    ]
+
+    total_issues, total_fixed, total_remaining = aggregate_tool_results(
```

Ruff (0.15.20)

[warning] Unpacked variable `total_fixed` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)